.tool-versions

```diff
@@ -1,2 +1,2 @@
-golang 1.23.0
-terraform 1.9.5
+golang 1.23.1
```

Comment:
given how the new toolchain works mixing with asdf can be cause issues so I'd recommend you use one or the other

Reply:
the thing is that asdf needs a specific version, 1.23 won't work, but I think 1.23 (not 1.23.1) is a best practice in go.mod.

what do you suggest to do?

Reply:
in the CLI we turned off the toolchain install feature https://github.com/mongodb/mongodb-atlas-cli/blob/0b91d7e5eb3cd1da233394a8e9b35cb6cc1ba593/Makefile#L39

Reply:
i see, if that's fine I'll keep it here as I think it's more handy for developers using asdf to have it